src/EnergyPlus/UnitarySystem.cc

```diff
@@ -12170,34 +12170,35 @@ namespace UnitarySystems {
                                                         this->m_SuppHeatPartLoadFrac);
         } break;
         case DataHVACGlobals::Coil_HeatingWater: {
-            // if (present(SuppCoilLoad)) {
+            // see if HW coil has enough capacity to meet the load
             if (SuppHeatCoilLoad > 0.0) {
-                // see if HW coil has enough capacity to meet the load
                 mdot = min(state.dataLoopNodes->Node(this->m_SuppCoilFluidOutletNodeNum).MassFlowRateMaxAvail, this->m_MaxSuppCoilFluidFlow);
```

Comment:
There is 2 parts to this simple change. The first is if there is a supplemental heating load set the water mass flow rate to max, else set it to 0.